src/pages/[platform]/build-a-backend/data/field-level-validation/index.mdx

```diff
@@ -0,0 +1,86 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+
+export const meta = {
+  title: 'Field level validation',
+  description: 'Learn how to enable field level validation in your model schema',
+  platforms: [
+    'android',
+    'angular',
+    'flutter',
+    'javascript',
+    'nextjs',
+    'react',
+    'react-native',
+    'swift',
+    'vue'
+  ]
+};
+
+export function getStaticPaths() {
+  return getCustomStaticPath(meta.platforms);
+}
+
+export function getStaticProps(context) {
+  return {
+    props: {
+      platform: context.params.platform,
+      meta
+    }
+  };
+}
+
+You can enable field level validation in your model schema by chaining a `validate` function to the field.
+
+## Examples
+
+```ts title="amplify/data/resource.ts"
+const schema = a.schema({
+  Todo: a.model({
+    content: a.string().validate(v => 
+      v
+        .minLength(1, 'Content must be at least 1 character long')
+        .maxLength(100, 'Content must be less than 100 characters')
+        .matches('^[a-zA-Z0-9\\\\s]+$', 'Content must contain only letters, numbers, and spaces')
+    )
+  })
+  .authorization(allow => [allow.publicApiKey()])
+});
+```
+
+## Supported validators
+
+### String Validators
+For `string` fields:
+
+| Validator | Description | Parameters | Example |
```

Comment:
regarding #8229 and similar efforts we should circle back to remove these tables once API reference is available